internal/config/config.go

```diff
@@ -69,16 +58,6 @@ func NewConfig(fileName FilePath) *Config {
 			ReadHeaderTimeout: getEnvInt("METRICS_READ_HEADER_TIMEOUT"),
 		},
 
-		LoggerConfig: &logging.Config{
```

Comment:
If we're gonna remove this then we should also remove these values from the 'config.env.template' file to avoid further regressions